OutOfSchool/OutOfSchool.WebApi.Tests/Controllers/StatisticControllerTests.cs

```diff
@@ -8,6 +8,7 @@
 using OutOfSchool.Services.Models;
 using OutOfSchool.WebApi.Controllers.V1;
 using OutOfSchool.WebApi.Models;
+using OutOfSchool.WebApi.Models.Workshop;
```

Comment:
(Redundant code) This `using` might be dropped

Reply:
And now I see that you moved it to other namespace. Previous comment is invalid, closing it